addons/im_livechat/static/src/core/common/transcript_sender.js

```diff
@@ -23,23 +21,47 @@ export class TranscriptSender extends Component {
 
     setup() {
         this.isValidEmail = isValidEmail;
-        this.livechatService = useService("im_livechat.livechat");
+        useAutofocus();
+        this.store = useService("mail.store");
         this.state = useState({
-            email: "",
+            email: this.initialEmail,
             status: this.STATUS.IDLE,
         });
     }
 
+    get initialEmail() {
+        return "";
```

Comment:
Couldn’t we get rid of this getter and its overrides by directly using the value of `props.thread.livechatVisitorMember.email`?